GVRf/Framework/src/org/gearvrf/GVRHybridObject.java

```diff
@@ -50,23 +51,41 @@
      */
     private long mNativePointer;
 
+    /**
+     * If true, the native object will be deleted by this class; otherwise,
+     * it won't be deleted.
+     */
+    private boolean mOwnNative;
```

Comment:
There is some assimp issue in the native version (not my code). However, the native class (VertexBoneData) is a data member of Mesh (not a pointer). So it does not need to be deleted. The existing impl. assumes all native objects need to be deleted, which is untrue in C++ case.